Update index.md

vidyavrat opened the pull request:
Made some changes to make content more meaningful and readable. Also removed 3rd paragraph which was duplicate of 1st one.

vidyavrat committed:
Update index.md

Made some changes to make content more meaningful and readable. Also removed 3rd paragraph which was duplicate of 1st one.

GuardRex commented on docs/framework/index.md:
```diff
@@ -26,11 +26,9 @@ manager: "wpickett"
 >  This .NET Framework content set includes information for .NET Framework versions 4.5, 4.5.1, 4.5.2, 4.6, 4.6.1, 4.6.2, and 4.7. To download the .NET Framework, see [Installing the .NET Framework](../../docs/framework/install/guide-for-developers.md). For a list of new features and changes in the NET Framework 4.5, the [!INCLUDE[net_v46](../../includes/net-v46-md.md)], their point releases, and ...
 
-The .NET Framework provides many services, including memory management, type and memory safety, security functionality, networking and application deployment. It provides easy-to-use data structures and APIs that abstract the lower-level Windows operating system. You can use a variety of programming languages with the .NET Framework, including C#, F# and Visual Basic.
+The .NET Framework provides many services, including memory management, type and memory safety, security, networking and application deployment. It provides easy-to-use data structures and APIs that abstract the lower-level Windows operating system. You can use a variety of programming languages with the .NET Framework, including C#, F# and Visual Basic .NET.
```
* Add a comma before "and" :point_right: "... networking, and application deployment."
* I recommend reverting the ".NET" there, since the passage is just listing the languages that can be used with the .NET Framework.

GuardRex commented on docs/framework/index.md:
```diff
@@ -26,11 +26,9 @@ manager: "wpickett"
- The .NET Framework is a development platform for building apps for Windows, Windows Phone, Windows Server, and Microsoft Azure. It consists of the common language runtime (CLR) and the .NET Framework class library, , which includes a broad range of functionality and support for many industry standards.  
-
-The .NET Framework provides many services, including memory management, type and memory safety, security functionality, networking and application deployment. It provides easy-to-use data structures and APIs that abstract the lower-level Windows operating system. You can use a variety of programming languages with the .NET Framework, including C#, F# and Visual Basic.
+ The .NET Framework is a development platform for building apps for Web, Windows, Windows Phone, Windows Server, and Microsoft Azure. It consists of the common language runtime (CLR) and the .NET Framework class library, , which includes a broad range of functionality and support for many industry standards.  
```
* Would you mind getting rid of that extra comma before "which" in the 2nd sentence?
* You can lowercase "Web" now :point_right: web

guardrex reviewed (changes requested):
Hello @vidyavrat ... thanks for helping update this topic. There are just a couple of minor updates to make.

vidyavrat committed:
Update index.md

@GuardRex please review the changes.

GuardRex commented:
@vidyavrat Perfect! Thanks! :smile: We'll get this merged after the build completes.

mairaw reviewed (changes requested):
Thanks for catching this and making the fixes as @GuardRex suggested. I have only one more request if you don't mind. Can you move the paragraph that you changed to where the repeated one was? I think that text should come after we talk about the kinds of apps you can build. Thank you!

GuardRex commented:
... and I missed a tiny nit ... that list at the end of the last sentence ... add a comma after F#. Thanks.

vidyavrat committed:
Update index.md

@mairaw please review the positioning of paragraphs and I think this will make sense for readers.

vidyavrat committed:
Update index.md

@guardrex please review.

GuardRex commented:
Thank you ... thank you. We're trying to get every little thing we can on these as we go. Future generations will thank us. 🌍

mairaw reviewed (approved):
Looks great! Thank you @vidyavrat!
